src/Lykke.Service.PayInvoicePortal/AutofacModule.cs

```diff
@@ -47,6 +50,23 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterRateCalculatorClient(_settings.CurrentValue.RateCalculatorServiceClient.ServiceUrl);
 
             builder.RegisterPayMerchantClient(_settings.CurrentValue.PayMerchantServiceClient, null);
+
+            builder.RegisterType<RealtimeNotificationsService>()
+                .As<IRealtimeNotificationsService>()
+                .SingleInstance();
+
+            RegisterRabbitSubscribers(builder);
+        }
+
+        private void RegisterRabbitSubscribers(ContainerBuilder builder)
+        {
+            builder.RegisterType<InvoiceUpdateSubscriber>()
+                .AsSelf()
+                .As<IStartable>()
```

Comment:
Registering it as IStartable potentially increases service start up time which will lead to fails while checking liveness probe. Consider another option - remove IStartable registration and start subscribers manually in StartupManager.

Reply:
fixed